woocommerce/woocommerce-ios
# Product Subscriptions: Add Free Trial and Expire After settings on Variable Subscriptions Product detail screen

Part of: #11171
Please do not merge until target is `trunk`



## Description


Similar to the simple subscription product counterpart, this PR enables the "Free Trial" and "Expire After" settings on Variable Subscription Product screen.

## Testing instructions


### Create a Variable subscriptions product:
1. Enable the feature flag subscriptionProducts and build the app.
2. On the app, log in to a store with Woo Subscription plugin set up.
3. Navigate to Products tab > select "+" > select manual option > Variable subscription product.
4. Notice that the product form displays similar fields as regular variable product's.
5. Create at least 1 attribute for the product and generate either one or all variations.
6. Save and publish

### Testing the settings:
1. Open the variable subscriptions product created above,
2. Tap "Variations",
3. Open one of the variation product,
4. Check and ensure the product detail shows both "Free Trial" and "Expire After",
5. Try out both settings, save and publish and make sure they work.

## Video


https://github.com/woocommerce/woocommerce-ios/assets/266376/1f300c06-167d-4a31-b0f9-929075cc625a

---
